Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'savon', '~> 2.12.0'
 gem 'sentry-raven'
 gem 'simple_command', '~> 0.0.9'
+gem 'state_machines-activerecord', '~> 0.5.2'
```

Comment:
why have you selected to use the gem `state_machines-activerecord` over [AASM](https://github.com/aasm/aasm)?

Reply:
The gem itself is [state_machines](https://github.com/state-machines/state_machines), `state_machines-activerecord` is the specific `activerecord` integration. As for the choice itself, there's a bunch of state machines gems out there such as:

- [state_machine](https://github.com/pluginaweek/state_machine)
- [state_machines](https://github.com/state-machines/state_machines)
- [aasm](https://github.com/aasm/aasm)
- [statesman](https://github.com/gocardless/statesman)
- [transitions](https://github.com/troessner/transitions)
- ...

I've used 3 or 4 out of the ones in the list before, and there's not many differences between most of them, the basic features I was expected to support:

* state machine definition
* helper methods such as transition methods and <state>?
* callbacks

are supported by it. 

So, the question is, is `AASM` (or any other gem) giving us anything else that **we need** and this one does support? If that's the case, I'm more than happy to change to it.